config.yaml

```diff
@@ -12,13 +12,13 @@ tft:
         type: "iperf-tcp"
         instances: 1
         server:
-          - name: "worker-229"
+          - name: "worker-235"
             persistent: "false"
             sriov: "true"
         client:
-          - name: "worker-230"
+          - name: "worker-236"
             sriov: "true"
         plugins:
           - name: capture_flows
-          - name: measure_cpu
-          - name: measure_power
+          - name: get_vf_rep
```

Comment:
Let's leave this as an example config file for users to reference with all the plugin options specified. If you only want to test w/ a subset, I would use an untracked personal config file